src/DocumentFormat.OpenXml/Packaging/OpenXmlPackage.FlatOpc.cs

```diff
@@ -14,10 +16,13 @@ namespace DocumentFormat.OpenXml.Packaging
     /// <summary>
     /// Represents a base class for strong typed Open XML document classes.
     /// </summary>
-    public abstract partial class OpenXmlPackage : OpenXmlPartContainer, IDisposable
+    public abstract partial class OpenXmlPackage
```

Comment:
I just realized you're removing class hierarchy - that's a breaking change. Why are you doing this?

Reply:
I don't remove class hierarchy. In a **partial class**, we don't have to mention the base class and interfaces again. I could add those back in if you like.

Reply:
Ah - did know it was defined on the other partial class. Yeah, I'm fine with not having it in this file (no need for duplication)